Lombiq.HelpfulLibraries/Libraries/Utilities/StringExtensions.cs

```diff
@@ -171,6 +172,40 @@ public static string OrIfEmpty(this string? text, params string?[] alternatives)
             Func<Task<string?>> alternativeAsync) =>
             condition(text) ? alternativeAsync() : Task.FromResult(text);
 
+        /// <summary>
+        /// Performs <see cref="Regex.Match(string, string, RegexOptions, TimeSpan)"/> with timeout (default is 1s).
+        /// </summary>
+        public static Match Match(

```

Comment:
Seems unnecessary to me. RegexReplace is only named like that only to avoid clashes with string.Replace. There is no such clash for Match and IsMatch.

Reply:
Yeah, but all of them proxy to `Regex.Something()`. "Match", when you're somewhere far from here and start typing `myString.M...`, seems ambiguous to me.

Reply:
I don't find it ambiguous becuase the concept of matching a string is already strongly associated with regex. I don't think there is anything in that context it can be ambiguated with.

Reply:
For some probably yes, but "match" just means some kind of search. `RegexReplace()` could also just be an overload of `Replace()` and it would similarly be an issue, but it has a distinction, which is a good thing.

Reply:
Guess we'll have to agree to disagree on this. I've reluctantly renamed them to `RegexMatch` and `RegexIsMatch`.

Reply:
Thank you, though I didn't want you to be reluctant :D.